src/cgutils.cpp

```diff
@@ -1937,18 +1937,22 @@ static jl_cgval_t typed_load(jl_codectx_t &ctx, Value *ptr, Value *idx_0based, j
             ctx.builder.CreateFence(Order);
         return ghostValue(ctx, jltype);
     }
+    if (isboxed)
+        alignment = sizeof(void*);
```

Comment:
I was under the impression that GC objects where 16 byte aligned?

Reply:
A pointer to a gc object only occupies a pointer sized space